tests/lapi/bitop_arshift_test.lua

```diff
@@ -0,0 +1,28 @@
+--[[
+SPDX-License-Identifier: ISC
+Copyright (c) 2023-2025, Sergey Bronnikov.
+
+Synopsis: bit32.arshift (x, disp)
+]]
+
+local luzer = require("luzer")
+local test_lib = require("lib")
+
+if test_lib.version() ~= "LuaJIT" then
+    os.exit(0)
+end
+
+local bitop = require("bit")
```

Comment:
The `bit` library is loaded by default in LuaJIT, there is no need to require it.
Here and below.

Reply:
Fixed, thanks!